deploy/datameta_with_proxy.yml

```diff
@@ -0,0 +1,106 @@
+version: '3.2'
```

Comment:
Maybe it would be good to add some hint to the filename or header to indicate that this is a docker compose config.